tests/decorators/decorators.go

```diff
@@ -27,4 +27,5 @@ var (
 	Reenlightenment      = []interface{}{Label("Reenlightenment")}
 	Invtsc               = []interface{}{Label("Invtsc")}
 	PasstGate            = []interface{}{Label("PasstGate")}
+	Istio                = []interface{}{Label("Istio")}
```

Comment:
maybe ISTIO ? if so all over of course please

Reply:
It's usually spelled Istio, so I would keep it as is